colorbleed/plugins/fusion/publish/submit_deadline.py

```diff
@@ -33,16 +40,20 @@ def process(self, context):
         # Collect all saver instances in context that are to be rendered
         saver_instances = []
         for instance in context[:]:
-            if instance.data.get("families")[0] != "fusion.deadline":
+            if not self.families[0] in instance.data.get("families"):
                 # Allow only saver family instances
                 continue
 
             if not instance.data.get("publish", True):
                 # Skip inactive instances
                 continue
-
+            self.log.debug(instance.data["name"])
             saver_instances.append(instance)
 
+        if not saver_instances:
+            self.log.warning(saver_instances)
```

Comment:
You are so interested in empty lists, ha! :) This doesn't tell us much I believe, let's remove the logged warning.